### DISCUSSION TITLE
Operating Systems supported by Salesforce CLI

### DISCUSSION THREAD

**Discussion by ikounalak** [forcedotcom/cli#2459] [Q&A] [answered]
As per documentation:

Salesforce CLI supports the following operating systems.
• Windows—Windows 8.1 and Windows Server 2012 (64-bit and 32-bit) or later
• Mac—macOS 10.11 or later (Intel and M1)
• Linux—Ubuntu 20.04

I would guess that Linux—Ubuntu 20.04 version is not restricted... we want to run cli on OS Red Hat Enterprise Linux release 8.8 (Ootpa) , any idea if that would cause an issue ?

**Comment by mshanemc** [comment, marked as answer]
I'm guessing it probably works.  Have seen other people use RHEL in our telemetry.

We define supported OS because those are ones that we run integration tests on for each CLI and plugin release.  We could also, for example, take an issue and replicate it on the supported OS.
